packages/client/bin/cli.ts

```diff
@@ -404,20 +404,20 @@ async function startClient(config: Config, customGenesisState?: GenesisState) {
     config.chainCommon.setForkHashes(blockchain.genesisBlock.hash())
   }
 
-  const client = new EthereumClient({
+  const client = await EthereumClient.create({
     config,
     blockchain,
     ...dbs,
   })
+  await client.open()
```

Comment:
This moving over was also necessary and I already thought "Oh would this have been the one-line fix already?" 🤪 but then I did the counter-test by using `master` and just moving the line and it actually didn't work (thanks god 😂) but broke with a `LEVEL_NOT_FOUND` error, actually another indicator that this new from-the-top-async-flow is more healthy to have things in place.